content/_index.md

```diff
@@ -516,6 +516,9 @@ versions:
 | Push Gateway API        | r0.1.0  |
 | Room Version            | v5      |
```

Comment:
Separate problem, but I don't really think that Matrix 1.0 corresponded to room v5: if you look at, say, https://github.com/matrix-org/matrix-spec-proposals/blob/f44483a8cf0f174376647ac5a777ad6145b5f195/specification/index.rst#specification-versions, it's clear that *all* room versions from v1 to v5 were part of Matrix 1.0.

So, maybe while we're here:

```suggestion
| Room Versions           | v1-v5   |
```

cc @turt2live, as the original author of this table

Reply:
at a spec level room versions aren't linear, so we'd define this as `1, 2, 3, 4, 5` (yes, v5 is also wrong - it's just `5`).

the backing history here is that it felt weird to list out the room versions like that, so a slight trick was applied: because all the room versions inherit from each other, even though they don't have to, we can simply point to the "latest" which drags the remainder into scope as well. The deeper rationale is that by listing fewer numbers, we made the table less scary. Since we're well past the 1.0 launch window though, we can probably reintroduce a lot of that complexity.

Reply:
```suggestion
| Room Versions           | 1, 2, 3, 4, 5 |
```

then

Reply:
Applied